server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -341,6 +435,34 @@ public List<SqlExecuteResult> getAsyncResult(@NotNull String sessionId, String r
         }
     }
 
+    public AsyncExecuteResultResp getAsyncResultV2(@NotNull String sessionId, String requestId) {
+        PreConditions.validArgumentState(Objects.nonNull(requestId), ErrorCodes.SqlRegulationRuleBlocked, null, null);
```

Comment:
hard to undertand here, why requestId == null means SqlRequlationRuleBlocked

Reply:
This is the old logic, it may be the logic agreed upon with the front-end.